### DISCUSSION TITLE
Change permissions in Dockerfile

### DISCUSSION THREAD

**Discussion by lucasdefrancesca** [AbsaOSS/spline-ui#250] [Q&A]
Hi, I'm trying to run the docker image (_absaoss-spline-web-ui:0.7.4_) on kubernets (OCP), but I wouldn't be able to, apparently because of an image permissions issue. Do you think that if it could be replicated as the rest-gateway Dockerfile has, it could work?

https://github.com/AbsaOSS/spline/blob/release/0.7.8/rest-gateway/Dockerfile

```dockerfile
USER 1001
COPY --chown=1001 target/*.war /usr/local/tomcat/webapps/ROOT.war
```

Error from kubernets POD:

```
java.io.IOException: Unable to create the directory [/usr/local/tomcat/webapps/ROOT]
at org.apache.catalina.startup.ExpandWar.expand(ExpandWar.java:116)
at org.apache.catalina.startup.ContextConfig.fixDocBase(ContextConfig.java:820)
at org.apache.catalina.startup.ContextConfig.beforeStart(ContextConfig.java:959)
```

Source:
https://developers.redhat.com/blog/2020/10/26/adapting-docker-and-kubernetes-containers-to-run-on-red-hat-openshift-container-platform

**Comment by wajda** [comment]
Created an issue for that - #251
Hence closing the discussion.
